<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1417</width>
    <height>818</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>261</width>
      <height>171</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Время системы (c)</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="time_system"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Шаг расчета</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="tima_calculation"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Масса системы</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="sum_mass_lineEdit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_27">
       <property name="text">
        <string>Импульс системы</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="impulse_lineEdit"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_28">
       <property name="text">
        <string>Момент импульса</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="impuls_moment_lineEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="formLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>0</y>
      <width>231</width>
      <height>170</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Масса</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="obj1_mass_lineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Координаты X</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="obj1_corX_lineEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Координаты Y</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="obj1_corY_lineEdit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Скорость X</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="obj1_speedX_lineEdit"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_24">
       <property name="text">
        <string>Скорость Y</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="obj1_speedY_lineEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="formLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>180</y>
      <width>231</width>
      <height>170</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_4">
     <item row="0" column="0">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Масса</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>Координаты X</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>Скорость X</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="obj2_mass_lineEdit"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="obj2_corX_lineEdit"/>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="obj2_speedX_lineEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>Координаты Y</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="obj2_corY_lineEdit"/>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="obj2_speedY_lineEdit"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_25">
       <property name="text">
        <string>Скорость Y</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="formLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>360</y>
      <width>231</width>
      <height>170</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_6">
     <item row="0" column="0">
      <widget class="QLabel" name="label_20">
       <property name="text">
        <string>Масса</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_21">
       <property name="text">
        <string>Координаты X</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_22">
       <property name="text">
        <string>Скорость X</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="obj3_mass_lineEdit"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="obj3_corX_lineEdit"/>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="obj3_speedX_lineEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_23">
       <property name="text">
        <string>Координаты Y</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="obj3_corY_lineEdit"/>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="obj3_speedY_lineEdit"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_26">
       <property name="text">
        <string>Скорость Y</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>0</y>
      <width>160</width>
      <height>103</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="save_Button">
       <property name="text">
        <string>Сохранить систему</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="load_Button">
       <property name="text">
        <string>Загрузить систему</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButtonStart">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>120</y>
      <width>158</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>120</y>
      <width>221</width>
      <height>30</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_32">
       <property name="text">
        <string>Количество звезд</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>0</y>
      <width>310</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_7">
     <item row="0" column="1">
      <widget class="QLineEdit" name="potencial_lineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_30">
       <property name="text">
        <string>Кинетическая энергия</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="kinetic_lineEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_31">
       <property name="text">
        <string>Полная энергия системы</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="full_energ_lineEdit"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_29">
       <property name="text">
        <string>Потенциальная энергия</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1417</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNEW">
   <property name="text">
    <string>NEW</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;new text&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
